src/integrationTest/resources/logback-test.xml

```diff
@@ -10,7 +10,9 @@
     <logger name="org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator" level="debug" additivity="true"/>
     <logger name="org.hibernate.SQL" level="debug" additivity="true"/>
     <logger name="org.hibernate.orm.jdbc.bind" level="trace" additivity="true"/>
+    <logger name="org.hibernate.orm.jdbc.extract" level="trace" additivity="true"/>
     <logger name="org.hibernate.persister.entity" level="debug" additivity="true"/>
+    <logger name="org.hibernate.orm.sql.ast.tree" level="debug" additivity="true"/>
```

Comment:
Enabling the `org.hibernate.orm.jdbc.extract` helps to make `ResultSet` process clear, just as `org.hibernate.orm.jdbc.bind` does for `PreparedStatement`;

Enalbing `org.hibernate.orm.sql.ast.tree` makes it possible to log Sql AST tree, which is based on visitor pattern as well, but only applicable to `select statement` or `mutation statement`, not to `table mutation`, so it is only enabled since this PR.